src/Microsoft.Azure.WebJobs.Logging/Internal/Utility.cs

```diff
@@ -72,5 +74,80 @@ public static string DecodeBase64(string base64)
             string str = Encoding.UTF8.GetString(bytes);
             return str;
         }
+
+
+        private static async Task SafeCreateAsync(this CloudTable table)
```

Comment:
Please add tests verifying this new code works. E.g. execute a write to a table that doesn't exist, etc.